CHANGES.rst

```diff
@@ -4,17 +4,10 @@ Changelog
 1.1.7 (unreleased)
 ------------------
 
-Breaking changes:
-
-- *add item here*
-
-New features:
-
-- *add item here*
-
```

Comment:
@bsuttor please keep these headers, when doing a release with plone.releaser they are automatically removed, but while it is still not there could be any potential break or new feature added and no header would be available.